integration-tests/integration_tests/integration_tests/component_tests/component_asynchronous_express_message_pattern_tests.py

```diff
@@ -206,3 +206,16 @@ def test_should_record_message_status_when_ebxml_error_response_returned_from_sp
                 'OUTBOUND_STATUS': 'OUTBOUND_SYNC_ASYNC_MESSAGE_SUCCESSFULLY_RESPONDED',
                 'WORKFLOW': 'sync-async'
             })
+
+    def test_should_return_bad_request_when_client_sends_invalid_message(self):
+        # Arrange
+        message, message_id = build_message('QUPC_IN160101UK05', '9689174606')
```

Comment:
dont need this, instead do
request_body = {'blah': '123'}

then use that in the act stage